-- trunk/libgdiplus/src/pngcodec.c	2010-07-07 21:08:33 UTC (rev 160025)
+++ trunk/libgdiplus/src/pngcodec.c	2010-07-07 21:14:27 UTC (rev 160026)
@@ -352,7 +352,7 @@
 						info_ptr->palette[i].blue,
 						info_ptr->palette[i].green,
 						info_ptr->palette[i].red,
-#if PNG_LIBPNG_VER > 10243
+#if PNG_LIBPNG_VER > 10399
 						info_ptr->trans_alpha [i]); /* alpha */
 #else
 						info_ptr->trans[i]); /* alpha */
@@ -422,7 +422,7 @@
 		}
 
 		if ((color_type == PNG_COLOR_TYPE_GRAY) && (bit_depth < 8)) {
-#if PNG_LIBPNG_VER > 10243
+#if PNG_LIBPNG_VER > 10399
 			png_set_expand_gray_1_2_4_to_8 (png_ptr);
 #else
 			png_set_gray_1_2_4_to_8(png_ptr);

